core/fse.js

```diff
@@ -825,10 +825,13 @@ exports.FullScreenEditorModule = exports.getModule = class FullScreenEditorModul
                             {
                                 const fromView = self.viewControllers.header.getView(MciViewIds.header.from);
                                 const area = getMessageAreaByTag(self.messageAreaTag);
-                                if(area && area.realNames) {
-                                    fromView.setText(self.client.user.properties[UserProps.RealName] || self.client.user.username);
-                                } else {
-                                    fromView.setText(self.client.user.username);
+
+				if(fromView !== undefined) {
```

Comment:
This isn't your doing, but just so you're aware there are newer APIs available that may help with some of this work in `menu_module.js` (available to all derived menus), and things like `client.user.getProperty()` to clean up some code as well.

Reply:
Views first, then modules. :D I'm still learning, hoping to do some more around this when I tackle documenting the modules. This one was just for my own board (which is not even anywhere close to live yet, I'm having too much fun working on the code to actually spend time getting it going yet :) ) One of the nice things about working on documentation though is besides learning a lot about how it works, I also find little stuff like this, and sometimes it's easier to fix it than document it. lol